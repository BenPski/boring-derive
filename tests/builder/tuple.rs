use boring_derive::Builder;

#[derive(Builder)]
struct Example(usize, String);

fn main() {}
