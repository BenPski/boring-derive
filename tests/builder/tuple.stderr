error: deriving builder pattern not supported for tuple structs
 --> tests/builder/tuple.rs:4:8
  |
4 | struct Example(usize, String);
  |        ^^^^^^^
